tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -829,7 +829,7 @@
   },
   {
     "id": "{{.ok}} deleted action {{.name}}\n",
-    "translation": "{{.ok}} deleted action {{.name}}\n"
+    "translation": "{{.ok}} deleted action '{{.name}}'\n"
```

Comment:
So I agree that the keys will be out of sync, but I think its weird and wrong they have the translation string as the key to begin with. Thats unnecessary and confusing in my opinion, just didnt want to go up and change them all to some other string which is more easily searchable. 

At the same time, boldness does not show on tools like Jenkins, travis, etc.  Only on the CLI, so there could still be a reason to quote them @mdeuser @dubeejw   

Although if you guys still think no, i can remove the quoting from the bolded strings.